daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Typing.scala

```diff
@@ -42,7 +42,7 @@ private[validation] object Typing {
   protected[validation] lazy val typeOfBuiltinFunction = {
     val alpha = TVar(Name.assertFromString("$alpha$"))
     val beta = TVar(Name.assertFromString("$beta$"))
-    val gamma = TVar(Name.assertFromString("$beta$"))
+    val gamma = TVar(Name.assertFromString("$gamma$"))
```

Comment:
@remyhaemmerle-da  This was a bug in the scala daml-lf typechecker that only affected the new MUL_NUMERIC and DIV_NUMERIC.